.vale.ini

```diff
@@ -15,10 +15,16 @@ BasedOnStyles = Vale, Apify, write-good, Microsoft
 TokenIgnores = (<\/?[A-Z][^>]*>), ([^\n]+=[^\n]*), ([^\n]+@[^\n]+\.[^\n]), ({[^}]*}), `[^`]+`
 # Ignore HTML comments and Markdown code blocks
 BlockIgnores = (?s) (<!--.*?-->)|(```.*?```)
-Vale.Spelling = YES
 
+# Enable Apify specific rules
+Apify.H1 = YES
+Apify.Apify = YES
+Apify.Capitalization = YES
+Apify.Languages = YES
```

Comment:
According to my local testing, these shouldn't be needed. The rules worked for me without this whole `# Enable Apify specific rules` section.